proto/osmosis/concentrated-liquidity/pool-model/query.proto

```diff
@@ -45,6 +45,12 @@ service Query {
     option (google.api.http).get =
         "/osmosis/concentratedliquidity/v1beta1/positions/{address}";
   }
+  // UserPositions returns all concentrated postitions of some address.
+  rpc TickLiquidityInBatches(QueryTickLiquidityInBatchesRequest)
+      returns (QueryTickLiquidityInBatchesResponse) {
+    option (google.api.http).get =
+        "/osmosis/concentratedliquidity/v1beta1/tick_liquidity_in_batches";
```

Comment:
getting this error when curling `http://localhost:1317/osmosis/concentratedliquidity/v1beta1/tick_liquidity_in_batches?pool_id=1&lower_tick=2&upper_tick=10`. I've also tried wrapping `lower_tick` and `upper_tick` with single and double quote, but got the same error:

```
{
  "code": 3,
  "message": "field type *types.Int is not supported in query parameters",
  "details": [
  ]
}
```

Reply:
also tried with the other endpoint `/osmosis/concentratedliquidity/v1beta1/liquidity_depths_for_range`, same error messages

Reply:
Fixed in https://github.com/osmosis-labs/osmosis/pull/4209/commits/14372a5abf09e4d22474e64c921c8b476bd3b282, changed from sdk.Int in the request to int64